circle.yml

```diff
@@ -14,7 +14,12 @@ dependencies:
   override:
     - $ANDROID_HOME/tools/android list target
     - make bootstrap-circle
-    - echo y | android update sdk --no-ui --all --filter android-23,build-tools-23.0.1,extra-android-m2repository,extra-google-m2repository,extra-android-support
+    - echo y | android update sdk --no-ui --all --filter android-25
```

Comment:
> Note: Only install one package at a time, as echo y will only work for one license. If you don’t do it this way, Android will give you an error message but won’t fail and your build continues. This is undesirable since it makes debugging very difficult.

from https://circleci.com/docs/android/#dependencies